docs/iiif_server_settings.md

```diff
@@ -37,6 +37,7 @@ On the IIIF Server Settings Form page, you will see the following:
 
 4. Checkbox to 'Enable IIIF Content Search API V1 and V2 endpoints'.
     - Checked by default in later (1.4.0+) versions of Archipelago.
+    - See the [related IIIF Manifest snippet shared here](iiif-content-search.md#1-iiif-manifest-templates)
```

Comment:
How's this @DiegoPino ? Let me know if you have specific language you'd like to see here instead.

Reply:
I think the language is Ok. Maybe just instead of "related" "needed" ? Content Search API won't work without it. But I am being too fine here. All good really